sodetlib/operations/translate_tes_parameters.py

```diff
@@ -0,0 +1,569 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import curve_fit
+from sodetlib.operations.iv import IVAnalysis
+from sodetlib.operations.bias_steps import BiasStepAnalysis
+from tqdm.auto import trange
+
+
+import warnings
+warnings.filterwarnings('ignore')
+np.seterr(all="ignore")
+
+
+def translate_tes_params(iv_file, bs_file, save_iv=False, save_bs=False):
```

Comment:
Also you don't need to worry about saving files, so I'd remove the args and all logic pertaining to saving the data.